src/opera/commands/deploy.py

```diff
@@ -45,6 +49,8 @@ def deploy(args):
         print("{0} is not a positive number!".format(args.workers))
         return 1
 
+    verbose_mode = True if args.verbose else False
```

Comment:
`args.verbose` is already a boolean, so no need for conversion here.

Reply:
Ok, I've removed that.